### DISCUSSION TITLE
Fine-tuning OWSM V2 with E-Branchformer Encoder

### DISCUSSION THREAD

**Discussion by Papazola43** [espnet/espnet#5536] [Q&A] [answered]
Hello,

I recently found that ESPnet has shared OWSM V2's parameters with E-Branchformer encoder and decided to fine-tune this OWSM V2 model with AISHELL. However, I am having trouble finding the training configuration file, which I believe is named "train_s2t_ebf_conv2d_size1024_e12_d12_piecewise_lr3e-4_warmup60k_flashattn.yaml". Could you please guide me on where I can obtain this configuration file?

Additionally, I would appreciate it if you could provide a step-by-step guide on how to fine-tune OWSM V2 with the AISHELL dataset, covering everything from data preprocessing to scoring, to ensure that I follow the correct procedure for the fine-tuning process.

Thank you very much

**Comment by sw005320** [comment]
Thanks for your report.
@pyf98, can you answer for it?

> Additionally, I would appreciate it if you could provide a step-by-step guide on how to fine-tune OWSM V2 with the AISHELL dataset, covering everything from data preprocessing to scoring, to ensure that I follow the correct procedure for the fine-tuning process.

This is a very good idea.
Thanks for the suggestion.
We'll work on it.

**Comment by pyf98** [comment]
OK, I will provide a guide on fine-tuning this model.

**Comment by pyf98** [comment, marked as answer]
Hi, here are some instructions about fine-tuning OWSM: https://github.com/espnet/espnet/pull/5539

Note: The model might easily overfit to a small training set. The fine-tuning config in the guide is not tuned.

**Comment by Papazola43** [comment]
Hi!
I've seen the instructions, and they are clear and easy to understand.
Thank you very much!
